### PR TITLE
Fix compatibility with PHP 7.2

### PR DESCRIPTION
**Pull request by martinvenus**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,8 @@ php:
   - 5.5
   - 5.6
   - 7.0
+  - 7.1
+  - 7.2
```
Comment (enumag) [inline .travis.yml]: Change the PHP version to use for code coverage to 7.2 as well. See line 30.

**Comment by enumag** [comment]
@martinvenus Feel free to remove support for older PHP versions if needed.

**Comment by martinvenus** [comment]
Done.
Removed: Nette 2.3 (does not have Nette\SmartObject), PHP 5.4 and 5.5

**Comment by enumag** [comment]
So you removed PHP 5.6 as well in the end huh? A bit inconsistent with other Kdyby packages but I don't think it matters too much. Is there anything else you want to do in this PR?

**Comment by martinvenus** [comment]
That's all I wanted to solve. Just please check carefully RedisSessionHandler test. There I was not 100 % sure. Thanks.

**Comment by enumag** [comment]
I noticed that... PHP sometimes changes the serialization format between verions. This is one of those cases probably.

**Comment by martinvenus** [comment]
Should I change something? Is there any problem?

**Comment by martinvenus** [comment]
We should probably update also https://github.com/Kdyby/Redis/blob/master/composer.json

**Comment by martinvenus** [comment]
composer.json updated.

**Comment by enumag** [comment]
Since you're updating this library to Nette 2.4, could you check [this PR](https://github.com/Kdyby/Redis/pull/69) if there is anything else we might want to have here?

**Comment by martinvenus** [comment]
There are some fixes solved better, so I will use some parts. I will do it in the evening. Thanks.

**Comment by martinvenus** [comment]
Done.

**Comment by enumag** [comment]
Thank you!

**Comment by enumag** [comment]
@martinvenus Did you test the master branch in a real application? I'd like it to be tested a little before I tag it.

**Comment by martinvenus** [comment]
I'm just testing with our application. I will give you know as soon as I finish. I haven't found any problem yet.

**Comment by enumag** [comment]
Thanks. Let me know when you're satisfied with the testing.

**Comment by martinvenus** [comment]
It would be really good if someone else also could test it.

**Comment by martinvenus** [comment]
I think that it's time to tag it :) Thanks.

**Comment by solcik** [comment]
So far so good.

**Pull request merged**